php/savesettings.php

```diff
@@ -48,6 +48,36 @@ function validDomain($domain_name)
 			"8.20.247.20" => "Comodo"
 		];
 
+$adlistsdefault = [];
+$adlistsuser = [];
+function readAdlists(&$list, $listname)
+{
+	// Reset list
+	$list = [];
```

Comment:
Why take in a list and just rewrite it? Could this return a list instead?
`$adlistsuser = readAdlists("user");`